recipe/meta.yaml

```diff
@@ -25,28 +24,39 @@ requirements:
     - setuptools_scm
   run:
     - python
-    - fancycompleter
-    - wmctrl
+    - fancycompleter >=0.8
```

Comment:
Upstream is using a fork of fancycompleter at the moment. Since it has no releases, it is unclear how we could add it to conda-forge. 
```python
"fancycompleter @ git+https://github.com/pdbpp/fancycompleter@master#egg=fancycompleter",  # noqa: E501
```

In short-term, I am calling upstream's tests which are currently passing.